papaparse.js

```diff
@@ -519,8 +523,21 @@ License: MIT
 
 			// Rejoin the line we likely just split in two by chunking the file
 			var aggregate = this._partialLine + chunk;
+			this._skipLines = parseInt(this._config.skipFirstNLines) || 0;
+			this._pendingSkip = parseInt(this._config.skipFirstNLines) || 0;
```

Comment:
I think we do not need to have  the `_skipLines` variable but just the pendingSkip and deacrese it each time. 
When pendingSkip is lower or equal than zero we should not skip anything else. 

This will make the code simpler.